### ISSUE TITLE
incompatible trace format between 4.9.0 and 4.8.0 

### ISSUE THREAD

**Issue by yuz10** [apache/rocketmq#2996]
**BUG REPORT**

1. Please describe the issue you observed
- What did you do (The steps to reproduce)?
1. consume message using 4.9.0, and send message trace.
2. view message trace in roctetmq-console, that uses 4.8.0

- What did you expect to see?
view message trace normally

- What did you see instead?
![image](https://user-images.githubusercontent.com/14816818/121755221-39573600-cb49-11eb-8bbb-72c1c49d2955.png)
```
[2021-06-12 06:34:28.197] ERROR op=global_exception_handler_print_error
java.lang.ArrayIndexOutOfBoundsException: 8
	at org.apache.rocketmq.console.util.MsgTraceDecodeUtil.decoderFromTraceDataString(MsgTraceDecodeUtil.java:106)
	at org.apache.rocketmq.console.model.MessageTraceView.decodeFromTraceTransData(MessageTraceView.java:50)
	at org.apache.rocketmq.console.service.impl.MessageTraceServiceImpl.queryMessageTraceByTopicAndKey(MessageTraceServiceImpl.java:47)
	at org.apache.rocketmq.console.controller.MessageTraceController.viewTraceMessages(MessageTraceController.java:71)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
```

2. Please tell us about your environment:

3. Other information (e.g. detailed explanation, logs, related issues, suggestions how to fix, etc):
![image](https://user-images.githubusercontent.com/14816818/121755451-c0a4a980-cb49-11eb-9f72-2df2ec7bb941.png)

**Comment by zongtanghu** [comment]
This console's messaage trace issue will be fix in the rocketmq 4.9.1 version.

**Comment by vongosling** [comment]
@duhenglucky label should be on pr not issue:-)
